app/models/offer_condition.rb

```diff
@@ -0,0 +1,20 @@
+class OfferCondition < ApplicationRecord
+  belongs_to :offer
+  has_one :application_choice, through: :offer
+
+  enum status: {
+    pending: 'pending',
+    met: 'met',
+    not_met: 'not_met',
+  }
+
+  audited associated_with: :application_choice
+
+  before_create :set_initial_status
+
+private
+
+  def set_initial_status
+    self.status ||= 'pending'
```

Comment:
This lets us specify `null: false` in the DB. Any issues with this we can forsee?

Reply:
We can set a default value in the database migration to avoid this